spec/system/provider_interface/provider_edits_condition_statuses_spec.rb

```diff
@@ -0,0 +1,102 @@
+require 'rails_helper'
+
+RSpec.feature 'Confirm conditions met' do
+  include CourseOptionHelpers
+  include DfESignInHelpers
+  include ProviderUserPermissionsHelper
+
+  before { FeatureFlag.activate(:individual_offer_conditions) }
+
+  scenario 'Provider user confirms offer conditions have been met by the candidate' do
+    given_i_am_a_provider_user_with_dfe_sign_in
+    and_i_am_an_authorised_provider_user
+    and_i_can_access_the_provider_interface
+
+    when_i_navigate_to_an_offer_accepted_by_the_candidate
+    and_i_navigate_to_the_offer_tab
+    and_click_on_confirm_conditions
+    then_i_should_see_a_summary_of_the_conditions
+
+    when_i_select_they_have_met_all_the_conditions
```

Comment:
Can we rename this to say we are only marking one condition as met rather than all?